src/components/dashboard/FaceVerification/api/FaceVerificationApi.js

```diff
@@ -42,8 +43,14 @@ class FaceVerificationApi {
       return sessionId
     } catch (exception) {
       const { message } = exception
+      const logArgs = ['Session token issue failed:', message, exception]
+
+      if (shouldLogVerificaitonError(exception)) {
```

Comment:
this needs to be discussed with @Esfir-E 
also we MUST log all unexpected errors
reason for FV failure is TBD with @Esfir-E

Reply:
@sirpy we have a lot of user-side related issues (e.g. dup found, bad image quality) on Sentry. There're no reason to log them

Reply:
it also logs it on amplitude